code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -418,6 +418,41 @@
 	new /obj/effect/decal/cleanable/blood/oil(loc)
 	..()
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/ed209/disassemble()
+	walk_to(src,0)
+	var/turf/Tsec = get_turf(src)
```

Comment:
Make this `snake_case` too please!

Reply:
Were you talking about the `Tsec` here as well ?

Reply:
Yup! Just any variable that doesn't really follow `snake_case` or is kind of cryptic, so any `Tsec` falls under it!

Reply:
Okay! That file was changed already. I finish with the floorbots dropping empty stack of tiles and then I'll do the ED209's Sa

Reply:
All `explode()` code `Tsec` have been renamed to `explode_turf`